packages/app/src/components/DBRowSidePanel.tsx

```diff
@@ -457,6 +459,8 @@ export default function DBRowSidePanelErrorBoundary({
   // Keep track of sub-drawers so we can disable closing this root drawer
   const [subDrawerOpen, setSubDrawerOpen] = useState(false);
 
+  const { isChildModalOpen } = useContext(RowSidePanelContext);
```

Comment:
Is this just to prevent closing of the sidebar when the code modal is open? 

If so, I think for simplicity's sake, we should just remove this logic.

Reply:
Ah got it. Yeah that's an unfortunate side effect of how the auto closing of the sidepanel works. Maybe we can create a follow-up to have a better pattern for that overall.?